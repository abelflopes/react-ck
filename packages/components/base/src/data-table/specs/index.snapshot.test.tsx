import React from "react";
import { DataTable } from "../index";
import renderer from "react-test-renderer";

describe("snapshot DataTable", () => {
  it("renders correctly", () => {
    const data = [{ a: 1, b: 2, c: 3 }];
    const tree = renderer.create(<DataTable data={data} data-testid="table" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
