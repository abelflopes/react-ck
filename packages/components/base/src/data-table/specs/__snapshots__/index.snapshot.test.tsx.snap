// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`snapshot DataTable renders correctly 1`] = `
<DocumentFragment>
  <div
    class="root spacing_default"
  >
    <table
      class="table"
      data-testid="table"
    >
      <tbody>
        <tr>
          <td>
            1
          </td>
          <td>
            2
          </td>
          <td>
            3
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</DocumentFragment>
`;
